src/Router.tsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
-import Home from './containers/Home';
-
+import Home from './containers/Home/Home';
+import NotFound from './containers/NotFound/NotFound';
+import Login from './containers/Login/Login';
 
 const Routes: React.FC = () => {
   return (
     <Switch>
         <Route path="/" exact component={Home} />
```

Comment:
mix of 2 and 4 space indents. we use prettier to machine format the code to reduce noise for code review.
there are many ways to use prettier, one of the way is to use a vscode extension https://github.com/prettier/prettier-vscode